### ISSUE TITLE
Roadmap to 1.0.0 release!

### ISSUE THREAD

**Issue by jwjeffr** [Chicago-Club-Management-Company/aqua-blue#52]
We're close to the 1.0.0 release! Roadmap items:

## Features

- Call `logging` inside the library! So users can collect logging info. It would be nice to include things like training accuracy in here.
 
## CI-CD changes:

- Only publish to `PyPI` on releases, not pushes
- Something like code coverage
- Docs should also be output as a `pdf` format for those that prefer a manual. This can be a file hosted directly in the repo.

## Advanced examples:

- Predicting a PDE
- Custom reservoir computing system using another library like `pytorch` for GPU support. Something like a basic multi-layer perception from `sklearn` would probably be ideal.
- Maybe a classification task? Create reservoir states from input signal and classify based on reservoir states. See [speech_commands](https://www.tensorflow.org/datasets/catalog/speech_commands) dataset.
- Couple reservoir with knowledge-based model. Double pendulum might be nice for this. We can also create pretty animations with this example
